Documentation/spelling_wordlist.txt

```diff
@@ -698,6 +698,7 @@ routable
 routingMode
 runnable
 runtimes
+sServiceHostRef
```

Comment:
... and also you can get rid of this, if you drop the explicit `k8sServiceHostRef` in the value description.

Reply:
Oh I see that was Sebastian adding this bit, please let's update anyway if you don't mind.

Reply:
Thanks for the suggestion, Quentin! I'll undo my change and update it according to your suggestion then.